proposal/color-4-new-spaces-js.d.ts.md

```diff
@@ -512,13 +524,13 @@ as the result of changing some of [`internal`]'s components.
   })
   ```
 
-* If space equals `lch` or `oklch`, let `changedColor` be the result of:
+* If `space` equals `lch` or `oklch`, let `changedColor` be the result of:
 
   ```js
   SassColor({
```

Comment:
It occurs to me that these should probably be

```suggestion
  new SassColor({
```

Reply:
Addressed in https://github.com/sass/sass/pull/3704/commits/7308783af5646e553c35b4944216bcc78b781a44